app/services/EventReportService.scala

```diff
@@ -244,9 +244,9 @@ class EventReportService @Inject()(eventReportConnector: EventReportConnector,
     apiProcessingInfo(eventType, pstr, deleteEvent) match {
       case Some(APIProcessingInfo(apiType, reads, schemaPath, connectToAPI)) =>
         val resp = for {
-          newUserAnswers <- eventReportCacheRepository.getUserAnswers(externalId, pstr, Some(EventDataIdentifier(eventType, year, version.toInt, externalId)))
+          newUserAnswers <- eventReportCacheRepository.getUserAnswers(externalId, pstr, None)
           oldUserAnswers <- eventReportCacheRepository.getUserAnswers(
-            externalId, pstr + "_original_cache", Some(EventDataIdentifier(eventType, year, version.toInt, externalId))
+            externalId, pstr + "_original_cache", None
```

Comment:
This is incorrect. Event specific data that we end up with is being compared to initial event data that we had. That comparison adds "changed" flags to member objects, to let the API know a member has been changed.